Source/WebCore/Modules/mediastream/gstreamer/GStreamerWebRTCUtils.cpp

```diff
@@ -192,7 +192,7 @@ static inline RTCRtpCodecParameters toRTCCodecParameters(const GstStructure* rtc
         parameters.payloadType = *pt;
 
     if (auto mimeType = gstStructureGetString(rtcParameters, "mime-type"_s))
-        parameters.mimeType = mimeType.toString();
+        parameters.mimeType = String(mimeType.span());
```

Comment:
Why do we have to explicitly write `String()` here? Doesn't it compile without it?